### ISSUE TITLE
Question regarding custom graphs

### ISSUE THREAD

**Issue by johansettlin** [flask-dashboard/Flask-MonitoringDashboard#324]
Hi, I was wondering how to get my custom graphs to show any data?

I was playing around with this function
```
def every_ten_seconds():
    print(int(random() * 100 // 10))
    return int(random() * 100 // 10)


every_ten_seconds_schedule = {'seconds': 10}
dashboard.add_graph("Every 10 Seconds", every_ten_seconds, "interval", **every_ten_seconds_schedule)
dashboard.bind(app)
```

The graph is there but it does not show any data.

**Comment by FlyingBird95** [comment]
Hey Johan,

thanks for your issue. Can you provide me a bit more info? Such as, can you check that the database is populated?

I also noticed a small bug with the visualization. In the graph-options-panel, you can select the end date (which is prefilled with the current day). However it shows the data up to today (and thus not including today). Can you check if this is the case for you as well?

Your code snippet above looks file, so that can't be the issue.

Kind regards,

Patrick

**Comment by johansettlin** [comment]
Hi, the database seems to be populated:
<img width="329" alt="Screenshot 2020-04-27 at 12 12 53" src="https://user-images.githubusercontent.com/37585850/80361228-e0630a80-8880-11ea-9360-3ca4be484bf5.png">

It feels like there could be something wrong with the configuration file:
<img width="443" alt="Screenshot 2020-04-27 at 12 13 16" src="https://user-images.githubusercontent.com/37585850/80361306-fffa3300-8880-11ea-8bb6-e12b14363237.png">

This is all that I can see on the custom graph page:
<img width="1096" alt="Screenshot 2020-04-27 at 12 17 57" src="https://user-images.githubusercontent.com/37585850/80361450-3a63d000-8881-11ea-9f9b-2f9f4b81bd1a.png">

**Comment by johansettlin** [comment]
And thank you for your answer!!

**Comment by mircealungu** [comment]
@johansettlin : did you try to chance the end date for the graph in the UI to tomorrow?

**Comment by johansettlin** [comment]
@mircealungu and @FlyingBird95  Changing the end date was the issue, thank you very much for the help! Now I can play on!
Cheer!

**Comment by mircealungu** [comment]
@johansettlin Nice :) 
Would you mind quickly filling our survey?
   https://forms.gle/EwqkwUtxwwuGCE2L7

It would help us with our research and future improvements. 

Thanks,
M.

**Comment by mircealungu** [comment]
@FlyingBird95 I guess we open a new issue with the problem so we don't forget to fix it? Or we keep this one around?

**Comment by FlyingBird95** [comment]
We can close this issue once #326 is merged ;)

**Comment by mpizosdim** [comment]
same issue here but `customgraphdata` table isnt populated. Also having this log on the console everytime I add `dashboard.add_graph(...)`:


>No commit has been made, due to the following error: Working outside of application context.
> This typically means that you attempted to use functionality that needed
> to interface with the current application object in some way. To solve
> this, set up an application context with app.app_context().  See the
> documentation for more information.

Not sure if it related though.

**Comment by mircealungu** [comment]
@mpizosdim you're right, it's not the same problem as the original. 
It's maybe a problem in the function that you're passing as argument to the add graph (equivalent to `every_ten_seconds` in the OPs example.
